eyeshade/controllers/owners.js

```diff
@@ -187,8 +188,10 @@ v3.bulk = {
           entry = await publishers.findOne({ publisher: channelId })
           if (!entry) continue
 
-          await publishers.update({ publisher: channelId }, { $set: { owner: owner.owner_identifier } },
-                                  { upsert: true })
+          if ((entry.owner) && (cleanup.indexOf(entry.owner))) cleanup.push(entry.owner)
```

Comment:
`cleanup` starts as `[]` so isn't `cleanup.indexOf(entry.owner)` going to return false and prevent pushing of owners to cleanup every time?

Reply:
* updated my comment above

Reply:
that's a typo. fixed